core/domain/stats_services.py

```diff
@@ -488,6 +489,27 @@ def get_sample_answers(exploration_id, exploration_version, state_name):
         for submitted_answer_dict in sample_answers]
 
 
+def get_state_answers_stats(exp_id, state_name, test_only_min_frequency=None):
```

Comment:
Please add full docstrings with args/returns/etc. to all functions.

Reply:
Done.